packages/components/table/stories/table.stories.js

```diff
@@ -0,0 +1,102 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, array } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.table';
+
+storiesOf('ts-table', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const cols = array('cols', [
+			{
+				visibility: 'desktop',
+				type: 'document',
+				property: 'context',
+				size: 'large',
+				value: 'Context'
+			},
+			{ visibility: 'desktop', type: 'status', property: 'type', size: 'small', value: 'Type', display: 'left' },
+			{ visibility: 'desktop', property: 'participants', size: 'small', value: 'Participants' },
+			{ visibility: 'desktop', property: 'newActivity', value: 'New Activity' },
+			{
+				visibility: 'desktop',
+				property: 'lastActivity',
+				value: 'Last Activity',
+				size: 'medium',
+				display: 'right'
+			},
+			{ visibility: 'mobile', type: 'document', property: 'mobileContext' },
+			{
+				visibility: 'mobile',
+				property: 'mobileLastActivity',
+				renderFunction: function(item) {
+					const div = document.createElement('div');
+					div.style.padding = '10px';
+					div.style.whiteSpace = 'nowrap';
+					if (item.unread && item.unread > 0) {
+						const child = document.createElement('div');
+						child.innerText = item.unread;
+						child.style.backgroundColor = '#01b0ff';
+						child.style.borderRadius = '3px';
+						child.style.color = 'white';
+						child.style.padding = '5px';
+						child.style.fontSize = '11px';
+						child.style.fontWeight = '600';
+						div.style.display = 'flex';
+						div.style.alignItems = 'center';
+						div.appendChild(child);
+					} else {
+						div.innerText = item.lastActivity;
+					}
+					return div;
+				}
+			}
+		]);
+		const data = array('data', [
+			{
+				content: {
+					context: { title: 'Invoice #32131', subtitle: 'Tradeshift' },
+					type: { status: 'ok', text: 'Internal' },
+					participants: 100,
+					newActivity:
+						'Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose.',
+					lastActivity: 'Yesterday, 15:01',
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1192)